docs/api/features/volumes.md

```diff
@@ -17,3 +17,36 @@ different from Docker Inc's docker:
    specified (including those that this container is sharing from others) are
    ignored. Only volumes belonging to the specified container will be
    considered.
+
+## Experimental support for NFS shared volumes
+
+[DOCKER-793](https://smartos.org/bugview/DOCKER-793) implements the first pass
+for supporting NFS shared volumes. The NFS shared volumes feature is described
+in details by its corresponding [RFD
+document](https://github.com/joyent/rfd/blob/master/rfd/0026/README.md).
+
+To enable support for NFS shared volumes in Triton, run the following command
+line from the headnode:
+
+```
+sdcadm experimental volapi
+```
+
+This command will create a new core zone that runs the VOLAPI service, which
+implements the Volumes API. It will also enable the
+`experimental_nfs_shared_volumes` metadata property in SAPI.
+
+At this point, all `docker volume` commands are supported but only for the
+`'tritonnfs'` volume driver, which provies support for NFS shared volumes. Note
+that the `'tritonnfs'` volume driver needs to be specified in the `docker volume
+create` command for it to work.
```

Comment:
Good point, RFD states:

> The tritonnfs driver is the default driver on Triton

whereas my thought was that in order to avoid confusion between docker/dockers' default volume driver and a default driver with a different behavior, we'd require users to specify `tritonnfs`. If that doesn't make sense, I'll make `tritonnfs` be the default driver on Triton.

Thanks for bringing that up!